### PR TITLE
added entry for files in cpp docs

### PR DESCRIPTION
**Pull request by bhagyamudgal**
### Description

- Added entry for Cpp Files.

### Type of Change

- [x] Adding a new entry
- [ ] Editing an existing entry (fixing a typo, bug, issues, etc)
- [ ] Updating the documentation

### Checklist

- [x] My entry follows the Codecademy Docs style guide
- [x] I have performed a self-review of my own writing and code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have checked my entry and corrected any misspellings
- [x] All writings are my own

### Codecademy Username

If you want your Codecademy username be displayed in your entry (feature coming in October), make sure to have your Codecademy user profile linked to your GitHub:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the GitHub field, add your GitHub profile URL.

### CODE REVIEW

File: content/cpp/concepts/files/files.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+Title: "Loops"
```
Comment (sonnynomnom) [inline content/cpp/concepts/files/files.md:2]: Files, not Loops

File: content/cpp/concepts/files/files.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+CatalogContent:
+  - "learn-c-plus-plus"
+  - "paths/computer-science"
+---
+
+A file is an object on a computer that stores data, information, settings, or commands used with a computer program.
+
+The `fstream` library allows us to work with files in C++.
+
+To use the `fstream` library in C++ program, include both the standard `<iostream>` AND the `<fstream>` header files in C++ program:
```
Comment (sonnynomnom) [inline content/cpp/concepts/files/files.md:17]: Lowercase AND -> and.

Comment (sonnynomnom) [reply content/cpp/concepts/files/files.md:17]: in the C++ program:

File: content/cpp/concepts/files/files.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+```cpp
+#include <iostream>
+#include <fstream>
+```
+
+There are three classes included in the `fstream` library, which are used to create, write or read files:
+
+1. `ofstream`: Creates and writes to files.
```
Comment (sonnynomnom) [inline content/cpp/concepts/files/files.md:26]: Let's use bullet points instead of numbers.

**Review by sonnynomnom — CHANGES_REQUESTED** [review]
Left some changes!

**Review by sonnynomnom — APPROVED** [review]
LGTM!

**Comment by sonnynomnom** [comment]
Hi @bhagyamudgal,

I made some changes: https://github.com/Codecademy/docs/commit/ca4038ba9ba2832f2fcfd11bb3c7d7434d475dca

It was brought to my attention that the first sentence might not be yours. We are human so I understand, but please refrain from doing this in the future. Because this could potentially jeopardize all other 600 entries 🙏. I just fixed it though so we good.

**Comment by bhagyamudgal** [comment]
@sonnynomnom Ok I will take care of this next time.

**Pull request merged**